packages/paste-design-tokens/tokens/global/box-shadow.yml

```diff
@@ -218,6 +218,13 @@ props:
   shadow-border-bottom-inverse-new-weaker:
     value: "{!offset-0} {!offset-05} {!offset-0} {!palette-purple-70}"
     comment: Bottom shadow border for the sidebar beta badge
+  shadow-border-bottom-weak:
+    value: "inset {!offset-0} -{!offset-10} {!offset-0} {!offset-negative-05} {!palette-gray-30}"
+    comment: Weak bottom shadow border for tab container
+  shadow-border-bottom-inverse-weaker:
+    value: "inset {!offset-0} -{!offset-10} {!offset-0} {!offset-negative-05} {!palette-gray-80}"
+    comment: Weaker bottom shadow border for tab container on inverse backgrounds
+
```

Comment:
issue: Since these are inset and offsite differently than our usual shadow-borders, they should be named differently

Reply:
Yes, I like the idea of adding in the `offset-negative` somewhere in there

Reply:
ok ok actually, I think if you change the values to this it should give the same effect:

`inset {!offset-0} {!offset-negative-05} {!offset-0} {!offset-0} ...`

And we can rename the tokens to `shadow-inset-border-bottom-...`